### DISCUSSION TITLE
rcases doesnt work

### DISCUSSION THREAD

**Discussion by rodhirejab** [KaTeX/KaTeX#3512] [Q&A] [answered]
Currently I'm using katex in Ghost

I would like to have a right bracket, 

I wrote this as html, it doesnt work
$$
\begin{rcases}
   a &\text{if } b \\
   c &\text{if } d
\end{rcases}⇒…
$$

but the left bracket works.

\[	
x = \begin{cases}
   a &\text{if } b \\
   c &\text{if } d
\end{cases}
\]

I am using the code from here.

you can check it out over here
https://www.navigatenegative.com/p/6d51feba-80b5-43af-9608-9a66545c5cb1/

![image](https://user-images.githubusercontent.com/98081004/150293326-fdcfb361-dc97-4046-bc6b-822423f465ca.png)

**Comment by ronkok** [comment, marked as answer]
@rodhirejab I followed the link you supplied. That page uses KaTeX version 0.10.1. `rcases` was introduced in version 0.12.0.

The current versison is 0.15.2. You can download the current version (0.15.2) [here](https://github.com/KaTeX/KaTeX/releases). If you can get that page to update to the latest version, you should be fine.

**Reply by rodhirejab** [reply]
WOOT!!!
 
 IT WORKS!!! THANKS!!!! 
 How did you check that I was using the old version?
Sorry, I am still new in this kind of stuff.

For those who are wondering, I got to read the "readme " note to get this code for code injection to the website

```html
<!DOCTYPE html>
<!-- KaTeX requires the use of the HTML5 doctype. Without it, KaTeX may not render properly -->
<html>
  <head>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.15.2/dist/katex.min.css" integrity="sha384-MlJdn/WNKDGXveldHDdyRP1R4CTHr3FeuDNfhsLPYrq2t0UBkUdK2jyTnXPEK1NQ" crossorigin="anonymous">

    <!-- The loading of KaTeX is deferred to speed up page rendering -->
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.15.2/dist/katex.min.js" integrity="sha384-VQ8d8WVFw0yHhCk5E8I86oOhv48xLpnDZx5T9GogA/Y84DcCKWXDmSDfn13bzFZY" crossorigin="anonymous"></script>

    <!-- To automatically render math in text elements, include the auto-render extension: -->
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.15.2/dist/contrib/auto-render.min.js" integrity="sha384-+XBljXPPiv+OzfbB3cVmLHf4hdUFHlWNZN5spNQ7rmHTXpd7WvJum6fIACpNNfIR" crossorigin="anonymous"
        onload="renderMathInElement(document.body);"></script>
  </head>
  ...
</html>
```

**Comment by ronkok** [comment]
If a page has KaTeX running, you can get the version by typing `katex.version` into the console. You can reach the console via `Ctrl+Shift+I`.

I'm glad the response helped. Please consider marking marking it as "answer".
